### ISSUE TITLE
Library ‘(scheme base)’ exports ‘expand’ macro keyword

### ISSUE THREAD

**Issue by mnieper** [larcenists/larceny#747]
By R7RS, ‘expand’ is not exported by ’(scheme base)’. However:

```
$ ~/larceny/larceny -r7rs
Larceny v0.98 "General Ripper" (Mar  7 2015 01:06:26, precise:Linux:unified)
larceny.heap, built on Sat Mar  7 01:06:50 EST 2015

> expand

Syntax violation: expand

Reference to macro keyword out of context

Form: expand

Trace: 

  expand



Error: unhandled condition:
Compound condition has these components: 
#<record &who>
    who : expand
#<record &message>
    message : "Reference to macro keyword out of context"
#<record &syntax>
    form : expand
    subform : #f

Entering debugger; type "?" for help.
debug> 
```

**Comment by WillClinger** [comment]
Thank you for reporting that.  The ```run``` keyword probably has this same problem.

**Comment by WillClinger** [comment]
In Larceny v0.98, a library cannot define and export any of the following identifiers: ```for```, ```run```, ```expand```, ```meta```.

To my surprise, that bug seems to have disappeared in the current development version of Larceny.  I don't understand why it disappeared, and will keep this ticket open for a while to remind me of the mystery.

**Comment by WillClinger** [comment]
Mostly fixed by changeset 30b2960d842c9a017d34717c0560156085ea23e9

Libraries can now define and export ```for```, ```run```, ```expand```, and ```meta```, but still cannot define or export ```program``` or ```primitives``` (or ```export``` etc, but that's less serious).

With the partial fix, I'm downgrading this ticket's priority and will probably change the milestone eventually.

**Comment by WillClinger** [comment]
Fixed by changeset 582c6082f1bbdfadc29d61e9848037d837b01d63
